.github/dependabot.yml

```diff
@@ -9,3 +9,9 @@ updates:
   allow:
   - dependency-type: direct
   - dependency-type: indirect
+- package-ecosystem: "github-actions"
```

Comment:
I do not understand why the dependabot config change is part of this PR.

Reply:
no is not related to this PR.